src/main/java/nopol_examples/nopol_example_12/NopolExample.java

```diff
@@ -2,12 +2,13 @@
 public class NopolExample {
     public boolean isEmpty(java.util.List list) {
         java.util.ArrayList list2 = new java.util.ArrayList();
```

Comment:
This line (4) has been identified with a suspiciousness value of 8.57%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_12.NopolExampleTest#test_3`
</details>